src/engine/renderer/gl_shader.cpp

```diff
@@ -898,6 +898,7 @@ static bool IsUnusedPermutation( const char *compileMacros )
 		else if ( strcmp( token, "USE_PHYSICAL_MAPPING" ) == 0 )
 		{
 			if ( !glConfig2.physicalMapping ) return true;
+			if ( !glConfig2.deluxeMapping ) return true;
```

Comment:
You already enforced No deluxe -> No physical at the glconfig level so it would be nicer not to add it here (keeping a direct correspondence between glconfig2.physicalMapping and USE_PHYSICAL_MAPPING).

Reply:
Right, it looks like my brain had two different solutions for the same problem and my fingers implemented both. 🤣️

I removed that commit.